Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -356,18 +357,18 @@ final class NotificationsCenterCell: UICollectionViewCell {
         // Foreground and Background Container Constraints
 
         NSLayoutConstraint.activate([
-            swipeBackgroundFillView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: NotificationsCenterCell.swipeEdgeBuffer * 2.0),
+            swipeBackgroundFillView.leadingAnchor.constraint(equalTo: contentView.readableContentGuide.leadingAnchor, constant: NotificationsCenterCell.swipeEdgeBuffer * 2.0),
```

Comment:
I wonder if we could adjust the placement of this `readableContentGuide` check elsewhere so that the swipe actions go all the way to the edge of the screen? This would line up better with how the saved articles cells act. Also it would be good if the background color when selected / highlighted can extend to the very edges too. We might want to confirm with @carolynlimadeo on this first.

**Current PR:**
![IMG_1249 2](https://user-images.githubusercontent.com/3620196/161117998-272b3cac-7eda-48de-9152-b007d0795498.PNG)
![IMG_1248](https://user-images.githubusercontent.com/3620196/161118000-281a9aaf-35fc-4a27-b75c-f4400e96c1dc.PNG)
![IMG_1247](https://user-images.githubusercontent.com/3620196/161118001-b7512eee-5149-4e15-ab0b-3d3819d7b2b5.PNG)

**Ideally:**
![IMG_1246](https://user-images.githubusercontent.com/3620196/161117974-ebc6491f-e2db-43a1-8521-14c1dff43d61.PNG)
![IMG_1245](https://user-images.githubusercontent.com/3620196/161117980-e0552f37-b448-4782-8f1b-ead9830f65de.PNG)
![IMG_1244](https://user-images.githubusercontent.com/3620196/161117982-5a0b19e3-2cc5-4ef9-8179-1819772c698a.PNG)

Reply:
This has been fixed, thanks.

Reply:
@mcleinman I'm seeing something odd when I first load the screen - the `messageSummaryLabel` appears squished at first, then expands out to the proper width after 1 second. I didn't notice it in `main`, so I think this is related to the `readableContentGuide` changes. I'm testing this on an iPad Pro 11-inch, iOS 15.2. This was the weird thing I remember seeing when getting those screenshots too.

![out](https://user-images.githubusercontent.com/3620196/162461063-5923e936-6390-4799-a98d-16b6c29c9df5.gif)

Reply:
Good catch. I couldn't isolate why this was happening - tried a few different things. Thus, I went to your way of making the readable content guide work - that seems to work great, as well as fix this problem. Thank you!